docs/stdlib/datetime.rst

```diff
@@ -138,22 +137,21 @@ EdgeDB stores and outputs timezone-aware values in UTC.
 
 .. eql:type:: std::datetime
 
-
-    A timezone-aware type representing a moment in time.
+    Represents a timezone-aware moment in time.
 
     All dates must correspond to dates that exist in the proleptic Gregorian
     calendar.
 
-    :eql:op:`Casting <cast>` is a simple way to obtain a
-    :eql:type:`datetime` value in an expression:
+    :eql:op:`Casting <cast>` is a simple way to obtain a :eql:type:`datetime`
+    value in an expression:
 
     .. code-block:: edgeql
 
         select <datetime>'2018-05-07T15:01:22.306916+00';
         select <datetime>'2018-05-07T15:01:22+00';
 
-    Note that when casting from strings, the string should be in ISO
-    8601 format with timezone included:
+    When casting :eql:type:`datetime` from strings, the string must follow to
+    the ISO8601 format with a timezone included.
```

Comment:
```suggestion
    the ISO 8601 format with a timezone included.
```

Please add a space between ISO and number in all instances. It's easier to read with the space.